As of commons-lang 3.6 StringEscapeUtils, CharSequenceTranslator and
NumericEntityEscaper have moved to commons-text.

https://gitlab.com/pdftk-java/pdftk/-/issues/51
--- java/com/gitlab/pdftk_java/data_import.java.orig	2020-05-01 05:05:17.000000000 -0500
+++ java/com/gitlab/pdftk_java/data_import.java	2020-05-11 23:14:53.000000000 -0500
@@ -25,7 +25,7 @@
 import java.io.InputStream;
 import java.util.ArrayList;
 import java.util.Scanner;
-import org.apache.commons.lang3.StringEscapeUtils;
+import org.apache.commons.text.StringEscapeUtils;
 import pdftk.com.lowagie.text.Rectangle;
 import pdftk.com.lowagie.text.pdf.PRIndirectReference;
 import pdftk.com.lowagie.text.pdf.PdfArray;
--- java/com/gitlab/pdftk_java/pdftk.java.orig	2020-05-01 05:05:17.000000000 -0500
+++ java/com/gitlab/pdftk_java/pdftk.java	2020-05-11 23:15:18.000000000 -0500
@@ -776,7 +776,7 @@
   static void describe_missing_library(Throwable error) {
     System.err.println("Error: could not load a required library for this operation.");
     System.err.println(error);
-    System.err.println("Make sure that bcprov and commons-lang3 are installed and included in the");
+    System.err.println("Make sure that bcprov, commons-lang3 and commons-text are installed and included in the");
     System.err.println("classpath. See also https://gitlab.com/pdftk-java/pdftk/issues/2.");
   }
 };
--- java/com/gitlab/pdftk_java/report.java.orig	2020-05-01 05:05:17.000000000 -0500
+++ java/com/gitlab/pdftk_java/report.java	2020-05-11 23:15:09.000000000 -0500
@@ -31,9 +31,9 @@
 import java.util.Iterator;
 import java.util.List;
 import java.util.Set;
-import org.apache.commons.lang3.StringEscapeUtils;
-import org.apache.commons.lang3.text.translate.CharSequenceTranslator;
-import org.apache.commons.lang3.text.translate.NumericEntityEscaper;
+import org.apache.commons.text.StringEscapeUtils;
+import org.apache.commons.text.translate.CharSequenceTranslator;
+import org.apache.commons.text.translate.NumericEntityEscaper;
 import pdftk.com.lowagie.text.Rectangle;
 import pdftk.com.lowagie.text.pdf.PRStream;
 import pdftk.com.lowagie.text.pdf.PdfArray;
@@ -49,7 +49,7 @@
 
   // Escape both Xml and Unicode entities
   // see
-  // https://commons.apache.org/proper/commons-lang/javadocs/api-3.6/org/apache/commons/lang3/StringEscapeUtils.html#escapeXml-java.lang.String-
+  // https://commons.apache.org/proper/commons-text/javadocs/api-release/org/apache/commons/text/StringEscapeUtils.html#escapeXml10-java.lang.String-
   private static CharSequenceTranslator XmlUnicodeEscaper = null;
 
   static String OutputXmlString(String jss_p) {
